config/ecs/container-insights/otel-task-metrics-config.yaml

```diff
@@ -83,8 +86,9 @@ processors:
 exporters:
   awsxray:
   awsemf/application:
-    log_group_name: 'aws/ecs/containerinsights/{ClusterName}/application/metrics'
-    log_stream_name: '{TaskId}'
+    namespace: ECS/ContainerInsightsMetrics
+    log_group_name: '/aws/ecs/containerinsights/application/metrics'
+    log_stream_name: 'metrics'
```

Comment:
Based on the discussion with cloudwatch team, we should use ClusterName and TaskId for the log_group_name and log_stream_name. Rayhan submitted a code change for this, please check this pr:
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/1623